admin/src/js/controllers/edit-user.js

```diff
@@ -464,17 +471,20 @@ angular
       return asynchronousValidations
         .then(valid => {
           if (!valid) {
-            $scope.setError();
+            // $scope.setError();
+            $log.info("FIX ME ASYNC VALID")
```

Comment:
If I recall correctly the `setError` function clears the "processing" state so the loading spinner goes away, the buttons are enabled again, and maybe sets some styling on the form. We'll want to include that styling here too.